build/workshops.md

```diff
@@ -2,8 +2,8 @@
 
 Contract proxy kit and Safe apps as well as the Safe apps SDK have been presented at several events.
 
-* [ETHOnline 2020 - Introduction to building with Safe Apps SDK & Contract Proxy Kit](https://www.youtube.com/watch?v=YGw8WfBw5OI)
-* [TruffleCon 2020 - Leverage Smart Contract Accounts for Your Dapp](https://www.youtube.com/watch?v=J_3BpQGTLeI)
-* [ETHDenver 2021 - Technical Introduction to the Safe Apps SDK](https://www.youtube.com/watch?v=QG94GpCE0p0)
-* [Hello Decentralization 2021 - Technical Introduction to the Safe Apps SDK](https://www.crowdcast.io/e/hello-decentralization/49)
-
+- [ETHTokyo 2023 - How to Make a Group Wallet to Buy Things with Frens using Safe{Core} Account Abstraction SDK](https://www.youtube.com/watch?v=Vgfgr-oNLMk&t=8360s) ([slides](https://docs.google.com/presentation/d/1menJOy_Hk3dS89VmpDU_ZxJcM90Set3ozNRZgBYJWy8/edit))
```

Comment:
2022 was a year with where Safe did a lof of talks/workshops that are not included here

Reply:
Where can I find these talks/workshops?

Reply:
I was thinking about these two, but I was checking them again and some things have changed over time:

- How Safe modules work: https://www.youtube.com/watch?v=QdOfuxxXVBA
  Only thing here is that the slides have the old logo

- Overview on the Core SDK, modules and guards: https://www.youtube.com/watch?v=ppwE9GXC5MA
  However, the packages inside the Core SDK have changed, so not ideal. :(

Reply:
Added: https://github.com/safe-global/safe-docs/pull/196/commits/82901ed8ad0bd6b1cbd7e6fd6d491ac1afb54766